src/bin/cobalt/new.rs

```diff
@@ -436,6 +436,37 @@ fn prepend_date_to_filename(
     Ok(())
 }
 
+fn move_from_drafts_to_posts(
+    config: &cobalt_model::Config,
+    file: &path::Path,
+) -> Result<path::PathBuf> {
+    let mut posts = config.posts.clone();
+    posts.include_drafts = true;
+    let posts = posts.build()?;
+    let drafts = posts.drafts.clone();
```

Comment:
Why did this need to be `clone`d?

Reply:
Because the map in the if below is moving it :)

Reply:
Why not do `posts.drafts.as_ref().map ....`?

Reply:
because I'm a noob x)